### ISSUE TITLE
can you add another spinner type?

### ISSUE THREAD

**Issue by zohar1000** [t-ho/ngx-ui-loader#80]
Hi, I like your spinner and use it in various projects.
I want to ask if can add 2 more spinner, these are from the new type, which is the latest trend.

The spinners can be seen in this spinner examples page from [lukehaas.me](https://projects.lukehaas.me/css-loaders).
Those I refer to are:
1. the right bottom one.
2. the second from the right on the first row.

Thanks for any effort,
Zohar

**Comment by t-ho** [comment]
Hi @zohar1000,

Thank you for using the lib. I think you can add your own template. Please see the documentation [here](https://tdev.app/ngx-ui-loader#custom-template) or a working example on Stackblitz [here](https://stackblitz.com/edit/ngx-ui-loader-custom-template).

**Comment by zohar1000** [comment]
Thanks @t-ho,

I was aware of the option to add my own template before I opened this issue.

The reason I opened this issue is not for my own project, it is just to make this library, which I think is great, to be more
appealing for new projects that have a need for a spinner. In 2 projects I was working on recently we used another library
for a spinner just because the spinner type we wanted (which is the new style) was not included in this library right out of the box and we didn't have the time to get into adding templates.

On any case thanks for the response.
